Dockerfile

```diff
@@ -1,17 +1,17 @@
 # set base image (host OS)
-FROM python:3.8
+FROM jfloff/alpine-python:3.8-slim
```

Comment:
Is this a legit repo? Whats wrong with python:3.8-slim? Did you readpython:3.8-slim

Reply:
I did read that article. I couldn't figure out any specifics from it. Just one guy raging against the friction that alpine causes. Which is real but you dont get the benefits without the work when it comes to alpine. 

In regards to `jfloff`, he shaved another 20mb off `3.8-slim` and came up with a crude Dockerfile pattern for dealing with requirements.txt files so I went with it. Not ideal, probably better to build our own alpine image, but I was not looking to do the leg work for a language I don't use.

Reply:
I dont think 20mb is worth going to some bespoke image. Especially if you move to using a xar that will remove both your time spent at each image start, and the complexities of the requirements.txt